pkg/services/database/database.go

```diff
@@ -243,3 +243,21 @@ func (d *database) findPreloadWitLimitAndPageQuery(
 
 	return d.connectionRead.Table(table).Where(where).Limit(limit).Offset(page * limit)
 }
+
+func (d *database) Batch(rawQuery string) error {
```

Comment:
Why can't this method keep the default of the others by returning `response.IResponse` ?